test/server/util/twilio.js

```diff
@@ -18,14 +18,7 @@ describe('Twilio client methods', function() {
     expect(mutateCallConnectContact('CAcbbf06f666c72c51c59200de56ae54ff', idCollection)).to.deep.equal(returnUpdate('CAcbbf06f666c72c51c59200de56ae54ff', CONNECT_CONTACT_URL));
     done();
   });
-  it('should be able to mutate calls when disconnecting volunteers from contacts', (done) => {
-    const userId = 1;
-    const volunteerCallSid = 'CAcbbf06f666c72c51c59200de56ae54ff';
-    expect(hangUpContactCall(volunteerCallSid, userId))
-      .to.deep.equal(returnUpdate(volunteerCallSid, DISCNNECT_CONTACT_URL));
-    done();
-  });
-  // test for hang up
+  // test for hang up ?
```

Comment:
There really isn't a good way to test this other than asserting that we get a call object back from Twilio. Maybe a timestamp changes but that's not a good way to confirm in a unit test.